yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,125 @@
+description: >
+    Implement to manage VPD data in system.
+methods:
+    - name: WriteKeyword
+      description: >
+          A method to update the value of a keyword in a given VPD file. Keyword
+          can be updated on hardware or it can be updated on both hardware and
+          cache. To update on hardware the target should be 1 and to update on
+          both hardware and cache the target is 2.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Path of the FRU whose keyword need to be upadted.
+          - name: keyword
+            type: struct[string,string,array[byte]]
+            description: >
+                It is a tuple of record, keyword and value that is to be
+                updated.
+          - name: target
+            type: size
+            description: >
+                Target where the keyword needs to be updated.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
```

Comment:
why is invalid argument removed , it is needed  along with all types of error that is going to be used. ECC check failure should be optional as this could be used in a non ecc based system

Reply:
Please use only common.errors as in future we may move this API under xyz. We want error handling to be generic.

Reply:
updated